autobot/telegram/objects/poll.py

```diff
@@ -1,8 +1,8 @@
 from .base import BaseObject
-from .user import User
-from .poll import PollOption
 from typing import List
```

Comment:
we don't need use typing.List anymore, it was deprecated. For built-in types, we can just use the built-in class as the type,
for example instead of import List from type, we can just use list like `a:list = []`